Fakebook.Posts/Fakebook.Posts.Domain/Models/Comment.cs

```diff
@@ -13,10 +13,16 @@ public class Comment
         public DateTime CreatedAt { get; set; }
         public ICollection<string> Likes { get; set; }
 
+
+
+
+
         public Comment(string userEmail, string content)
         {
             if (string.IsNullOrWhiteSpace(userEmail)) throw new ArgumentException("User email is required.", nameof(userEmail));
             if (string.IsNullOrWhiteSpace(content)) throw new ArgumentException("Post content is required.", nameof(content));
+            if (!userEmail.Contains("@")) throw new ArgumentException("Post content is required.", nameof(content));
+
```

Comment:
this exception doesnt make sense.
and the whitespace being added above should not be there